src/functionalTest/java/uk/gov/hmcts/reform/waworkflowapi/controllers/SendMessageTest.java

```diff
@@ -302,4 +330,63 @@ public void should_not_be_able_to_post_as_message_does_not_exist() {
 
     }
 
+    private void createSpecifiedStandaloneTask(String taskName, String taskType, String caseType, String jurisdiction, String roleCategory) {
```

Comment:
Could this be put in its own class so it can be re-used?  Seems like it could be useful in other tests too.

Reply:
will work on it

Reply:
done